ping/.gitignore

```diff
@@ -0,0 +1,3 @@
+target/**/
```

Comment:
This file tells `git` to ignore certain files, in this case, IntelliJ files, and compilation results.